ansible/roles/stack-sunbird/templates/sunbird_player.env

```diff
@@ -113,4 +113,21 @@ reportsListVersion={{reportsListVersion|default('v1')}}
 
 #Release-3.2.7
 sunbird_data_product_service=http://kong:8000/
-discussions_middleware={{ discussionsmw_service_url | default('http://discussionsmw-service:3002') }}
\ No newline at end of file
+discussions_middleware={{ discussionsmw_service_url | default('http://discussionsmw-service:3002') }}
+
+#Release-3.6.0
+sunbird_kong_device_register={{sunbird_kong_device_register|default('false')}}
+sunbird_kong_device_register_token={{sunbird_kong_device_register_token|default('')}}
+
+# Allowed type sentinel/cluster/standalone(default)
+sunbird_redis_type={{sunbird_redis_type|default('standalone')}}
+# For sentinel and cluster use below.
+sunbird_redis_urls={{sunbird_redis_urls|default("redis.dev.svc.cluster.local")}}
+sunbird_redis_port={{sunbird_redis_port|default(26379)}}
```

Comment:
change the port to 6379
andd add a comment saying, in case of sentinel, add sentinel port (usually 2639)

Reply:
Updated